frontend/src/component/feature/FeatureStrategy/FeatureStrategyConstraints/ValueList.tsx

```diff
@@ -44,18 +50,82 @@ const ValueChip = styled(ValueChipBase)(({ theme }) => ({
     },
 }));
 
-const AddValuesButton = styled(ValueChipBase)(({ theme }) => ({
+const AddValuesButton = styled('button')(({ theme }) => ({
     color: theme.palette.primary.main,
     svg: {
         fill: theme.palette.primary.main,
         height: theme.fontSizes.smallerBody,
         width: theme.fontSizes.smallerBody,
     },
-    ':hover': {
+    border: 'none',
+    borderRadius: theme.shape.borderRadiusExtraLarge,
+    display: 'flex',
+    flexFlow: 'row nowrap',
+    whiteSpace: 'nowrap',
+    gap: theme.spacing(0.25),
+    alignItems: 'center',
+    paddingInline: theme.spacing(1.5),
+    height: theme.spacing(3),
+    transition: 'all 0.3s ease',
+    outline: `1px solid #0000`,
+    background: theme.palette.background.elevation1,
+    ':hover, :focus-visible': {
+        background: theme.palette.background.elevation1,
         outlineColor: theme.palette.secondary.dark,
     },
 }));
 
+const StyledPopover = styled(Popover)(({ theme }) => ({
+    '& .MuiPaper-root': {
+        borderRadius: theme.shape.borderRadiusLarge,
+        border: `1px solid ${theme.palette.divider}`,
+        padding: theme.spacing(1),
+    },
+}));
+
+const AddValues = forwardRef<HTMLButtonElement, {}>((props, ref) => {
+    const [open, setOpen] = useState(false);
+    const positioningRef = useRef<HTMLButtonElement>(null);
+    useImperativeHandle(ref, () => positioningRef.current as HTMLButtonElement);
+
+    return (
+        <>
+            <AddValuesButton
+                ref={positioningRef}
+                onClick={() => setOpen(true)}
+                type='button'
+            >
+                <Add />
+                <span>Add values</span>
+            </AddValuesButton>
+            <StyledPopover
+                open={open}
+                disableScrollLock
+                anchorEl={positioningRef.current}
+                onClose={() => setOpen(false)}
+                anchorOrigin={{
+                    vertical: 'bottom',
+                    horizontal: 'center',
+                }}
+                transformOrigin={{
+                    vertical: 'top',
+                    horizontal: 'center',
+                }}
+            >
+                <form action='' onSubmit={(e) => e.preventDefault()}>
+                    <input type='text' />
+                    <Button
+                        variant='text'
+                        onClick={() => console.log('clicked the add button')}
+                    >
+                        Add
+                    </Button>
+                </form>
```

Comment:
Yeh, yeh, this deosn't do anything yet. Also form submission prevention doesn't work right yet (it actually tries to save the parent form when you do this). But we'll tackle that later.